src/shared/onefx-auth-provider/email-password-identity-provider/email-password-identity-provider-handler.tsx

```diff
@@ -141,6 +142,18 @@ export function setEmailPasswordIdentityProviderRoutes(server: MyServer): void {
           locale,
           ip: ctx.headers["x-forwarded-for"],
         });
+
+        const userWorkspace = await server.model.workspaceModel.create({
+          name: workspaceName,
+          owner: user._id,
+        });
+
+        await server.model.userWorkspace.create({
+          workspace: userWorkspace._id,
+          user: user._id,
+          role: WorkspaceRole.follower,
```

Comment:
the owner is a admin I think...

Reply:
Tian says use "owner" , but there is not "owner" in workspaceRole . "owner" is "admin" ? @puncsky